src/jit/jitconfigvalues.h

```diff
@@ -0,0 +1,193 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if !defined(CONFIG_INTEGER) || !defined(CONFIG_STRING) || !defined(CONFIG_METHODSET)
+#error CONFIG_INTEGER, CONFIG_STRING, and CONFIG_METHODSET must be defined before including this file.
+#endif // !defined(CONFIG_INTEGER) || !defined(CONFIG_STRING) || !defined(CONFIG_METHODSET)
+
+#if defined(DEBUG)
+CONFIG_INTEGER(AltJitLimit, W("AltJitLimit"), 0) // Max number of functions to use altjit for (decimal)
+CONFIG_INTEGER(AltJitSkipOnAssert, W("AltJitSkipOnAssert"), 0) // If AltJit hits an assert, fall back to the fallback JIT. Useful in conjunction with COMPLUS_ContinueOnAssert=1
```

Comment:
@russellhadley - just because JIT sources are OSS doesn't mean that we have to support all config settings that are meant for internal use.  If really needed we can enforce in code that these are not available in official product builds.  We don't do that right now is another thing.

@pgavlin - I wouldn't worry about breaking users who are using undocumented and meant for internal usage

Having said that, my intent is that we capture which setting we intended to be internal and which is external to be captured in some form.  I thought adding an explicit field in the macro would force the users to think when they are adding a new config value rather than a comment.  But i will leave it to you.